packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# 49.12.0 - September 30, 2020
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Show progress bar when installing dependencies for Function Create ([PR #2563](https://github.com/forcedotcom/salesforcedx-vscode/pull/2563))
+
+- Add feature flag for Functions ([PR #2572](https://github.com/forcedotcom/salesforcedx-vscode/pull/2572))
+
+- Show message if no default org is set when starting Function ([PR #2560](https://github.com/forcedotcom/salesforcedx-vscode/pull/2560))
+
+#### salesforcedx-vscode-soql
```

Comment:
Same with soql - I don't believe we'll want to include anything in the changelog for these features yet